lib/peek_app_sdk/ui/core_components.ex

```diff
@@ -956,15 +956,13 @@ defmodule PeekAppSDK.UI.CoreComponents do
     ]}>
       <%= if @top_caret do %>
         <div class="w-0 h-0 border-l-[6px] border-l-transparent border-r-[6px] border-r-transparent border-b-[6px] border-b-black"></div>
+        <% else %>
```

Comment:
I think the project might be missing the phoenix formatter?
```suggestion
      <% else %>
```

Reply:
okay just fixed! will try to get the formatter working because running `mix format` didn't work either